src/sentry/conf/server.py

```diff
@@ -1099,8 +1099,14 @@ def create_partitioned_queues(name):
 SENTRY_METRICS_PREFIX = "sentry."
 SENTRY_METRICS_SKIP_INTERNAL_PREFIXES = []  # Order this by most frequent prefixes.
 
-# URI Prefixes for generating DSN URLs
-# (Defaults to URL_PREFIX by default)
+# URI Prefixes for generating DSN URLs.
+#
+# Custom schema/hostname/port to show for all DSNs. This can contain formatting
+# placeholders for organization and project ID:
+#
+#    https://p{project_id}.o{organization_id}.ingest.sentry.io
```

Comment:
Note that it will have to be either `sentry.io` or `o<id>.ingest.sentry.io` based on a feature flag on the organization. Of course, only Sentry installations that have multiple organizations enabled in settings. Relay will hardcode this.